queries/water-z11.pgsql

```diff
@@ -35,6 +35,24 @@ FROM
         AND way_area::bigint > 25600 -- 4px
         AND mz_way12 && !bbox!
 
+    --
+    -- Water line geometries
+    --
+    UNION
+
+    SELECT name,
+           NULL AS area,
```

Comment:
Good question. We have a column for the area in our tables, but I'm not sure how it's calculated. I was assuming that it would have been the line length, but let me run a query to see if there's actually any helpful data in the database.

If there isn't, then a condition in the where clause won't be helpful either.